fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -46,9 +46,9 @@ NONE_METHOD_OPT="NONE"
 # --------------------------------------------------------------------------------
 opts_SetScriptDescription "Run fMRIVolume processing"
 
-opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all subject" "--path"
+opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all sessions" "--path"
 
-opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddMandatory '--session' 'Session' 'Session ID' "--subject"
```

Comment:
Again, missing a long description argument, so it will not use the `--subject` argument as a legacy flag.